### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by FinalStarWhal**
Updated all headers test to not use netstat.

Updated README to fix readability and added a basic setup section.

### CODE REVIEW

**Comment by randall** [comment]
[approve ci]

File: tests/README.md [hunk 1]
```diff
@@ -1,38 +1,44 @@
...
-1. run the bootstrap script then source the path with a "source ./env-test/bin/activate" command. At this point autest command should run without the wrapper script
-2. The other way is to make sure you install python 3.5 or better on your system. From there install these python packages ( ie pip install ):
+AuTest and the relevant tools can be install manually instead of using the wrapper script. By doing this, it is often easier to debug issues with the testing system, or the tests. There are two ways this can be done.
+1. Run the bootstrap script then source the path with a "source ./env-test/bin/activate" command. At this point autest command should run without the wrapper script
+2. Make sure you install python 3.5 or better on your system. From there install these python packages ( ie pip install ):
   - hyper
-  - git+https://bitbucket.org/autestsuite/reusable-gold-testing-system.git
+  - git+ https://bitbucket.org/autestsuite/reusable-gold-testing-system.git
```
Comment (dragon512) [inline tests/README.md]: The git+ https:// ... cannot have any space in it.  needs to be git+https://

File: tests/README.md [hunk 2]
```diff
@@ -1,38 +1,44 @@
...
   - [traffic-replay](https://bitbucket.org/autestsuite/trafficreplay/src/master/) (This will automatically install [MicroDNS](https://bitbucket.org/autestsuite/microdns/src/master/), [MicroServer](https://bitbucket.org/autestsuite/microserver/src/master/), [TrafficReplayLibrary](https://bitbucket.org/autestsuite/trafficreplaylibrary/src/master/), and dnslib as part of the dependencies.)
 
-# Writting tests for AuTest
-When writting for the AuTest system please refer to the current documenation on the [online wiki](https://bitbucket.org/dragon512/reusable-gold-testing-system/wiki/Home) for general use of the system.
+# Writing tests for AuTest
+When writing for the AuTest system please refer to the current documenation on the [online wiki](https://bitbucket.org/dragon512/reusable-gold-testing-system/wiki/Home) for general use of the system.
```
Comment (dragon512) [inline tests/README.md]: Can you update the link to point to https://autestsuite.bitbucket.io/ and change "online wiki" to soemthing else.. this is sphinx documentation

**Comment by dragon512** [comment]
[approve ci]

**Comment by dragon512** [comment]
[approve ci]

**Comment by FinalStarWhal** [comment]
I'm going to look at why the test failed.

**Comment by dragon512** [comment]
[approve ci autest]

**Comment by dragon512** [comment]
looks like the failure was some posix lock issue at startup.. .some sort of race.

**Pull request merged**